data/json/proficiencies/electronics.json

```diff
@@ -9,6 +9,38 @@
     "default_fail_multiplier": 1.5,
     "time_to_learn": "5 h"
   },
+  {
+    "type": "proficiency",
+    "id": "prof_elec_circuit_theory",
+    "name": { "str": "Electrical Circuit Theory" },
+    "description": "An understanding of the basic principles of electrical circuits: continuity, current, voltage, resistance, and Ohm's law.",
```

Comment:
This sounds very theoretical. Aren't proficiencies supposed to be about hands-on stuff?

Reply:
Maybe "Theory" is a poor word choice (I am not an electrical engineer, only a former hobbyist). "Fundamentals" or "Principles" is more what I was going for here.

It does relate to hands-on knowledge like choosing the right components, wire gauge and power supply, knowing how to avoid electrocution when working with vehicle batteries, etc - but that practical knowledge stems from this more fundamental grasp of how electricity moves through a circuit. Perhaps this is too abstract for a proficiency, but I think it fits the precedent of other low-level "Principles" proficiencies we have.

Reply:
I changed this proficiency to "Electrical Circuits: Practical experience building electrical circuits, and a grasp of the principles of electrical continuity, current, voltage, and resistance."